models/iaf_psc_exp_htum.h

```diff
@@ -131,17 +132,22 @@ the NEST source code (docs/model_details).
        space analysis of synchronous spiking in cortical neural
        networks. Neurocomputing 38-40:565-571.
        DOI: https://doi.org/10.1016/S0925-2312(01)00409-X
-\endverbatim
 
-Sends: SpikeEvent
 
-Receives: SpikeEvent, CurrentEvent, DataLoggingRequest
+Sends
++++++
+
+SpikeEvent
+
+Receives
+++++++++
+
+SpikeEvent, CurrentEvent, DataLoggingRequest
 
 FirstVersion: March 2006
 
-Author: Moritz Helias
-*/
-class iaf_psc_exp_htum : public Archiving_Node
+EndUserDocs */
+class iaf_tum_2000 : public Archiving_Node
```

Comment:
Something's fishy here. Why is there a model name change in this PR? Is this a mixup with changes from another branch? I think the correct name should be `iaf_psc_exp_htum`.

Anyway, the (incomplete) change of name leads to the failure of the Travis tests. This should be sorted out.